src/bt_peer_connection.cpp

```diff
@@ -348,6 +348,17 @@ namespace {
 		stats_counters().inc_stats_counter(counters::num_outgoing_dht_port);
 	}
 
+	template<class F, typename... Args>
+	void bt_peer_connection::extension_notify(F message, Args... args)
+	{
+#ifndef TORRENT_DISABLE_EXTENSIONS
+		for (auto const& e : m_extensions)
+		{
+			(*e.*message)(args...);
```

Comment:
oh, I guess since we make multiple calls here, we can't move the arguments. so, never mind. they're all primitive types anyway